slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbLocker.java

```diff
@@ -17,13 +17,23 @@
 public class DbLocker {
 
   public enum LockType {
-    BLOCK,
-    ATTESTATION
+    BLOCK(0),
+    ATTESTATION(1),
+    ;
```

Comment:
This formatting is a bit nasty, I'm guessing this spotless enforcing this though?

Reply:
Let me see if I am able to remove that `;`.

Reply:
removed ;